src/helpers/dataUrl.js

```diff
@@ -0,0 +1,10 @@
+export const endpoint = 'https://api.shorten.rest/aliases';
+export const data = {
```

Comment:
The name `helpers` usually means functions. But you use it as constants. 

Basically, I would expect a function which takes url and returns an object for future request.